package.json

```diff
@@ -44,7 +44,7 @@
     "commands": [
       {
         "command": "code-runner.run",
-        "title": "Run Code",
+        "title": "Run with Code Runner",
```

Comment:
I think we'd better keep the original title.

Reply:
How about create a new command but with the same execution logic. The new command is only for `editor/title/run`?